src/Voron/Data/CompactTrees/CompactKey.cs

```diff
@@ -22,22 +18,17 @@ public sealed unsafe class CompactKey : IDisposable
 {
     public static readonly CompactKey NullInstance = new();
 
-    [ThreadStatic]
-    private static ArrayPool<byte> StoragePool;
-    [ThreadStatic]
-    private static ArrayPool<long> KeyMappingPool;
```

Comment:
Can you please elaborate on this change? I remember we had some issues in the past when that piece of code was modified.